service-go/cmd/service/service.go

```diff
@@ -2,9 +2,10 @@ package service
 
 import (
 	"fmt"
-	"log"
 	"net"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
I believe logrus needs to be initialized (you would get a runtime error). So instead of renaming "logrus" as "log", instantiate a logger using
```go
log := logrus.New()
```

Reply:
This came straight from logrus README, please check out https://github.com/sirupsen/logrus#example

Creating logger instances is an option, but not necessary. It's recommended if we need `logging to multiple locations from the same application` 
Let's decide which one works best for our case during the interview.